django/db/backends/dummy/base.py

```diff
@@ -83,4 +83,4 @@ def __init__(self, *args, **kwargs):
         self.validation = BaseDatabaseValidation(self)
 
     def is_usable(self):
-        return True
+        return False
```

Comment:
I guess those exceptions would be reraised later on at some point though. I think the consistent history checks aren't  so critical that some small chance of incorrectly skipping them will cause big problems (after all, we've had migrations since 1.7 and the chance of inconsistent history is fairly low in the first place).

Unless someone has an alternate proposal that isn't too invasive, I'm inclined to go with the exception catching that I proposed. We might revisit the issue in 1.11 if needed.